cs2log_test.go

```diff
@@ -1,22 +1,25 @@
-package cs2log
+package cs2log_test
 
 import (
 	"fmt"
 	"regexp"
+	"strconv"
 	"strings"
 	"testing"
 	"time"
+
+	cs2log "github.com/janstuemmel/cs2-log"
```

Comment:
no need for naming here as the packages name is `cs2log`

Reply:
Oops sorry, my auto-correct did it wrong.

Reply:
the same with my ide, so its ok i guess